bratshelper/tests.go

```diff
@@ -254,7 +254,7 @@ func StagingWithCustomBuildpackWithCredentialsInDependencies(depRegexp string, c
 				buildpackFile = Data.UncachedFile
 			})
 			It("does not include credentials in logged dependency uris", func() {
-				Expect(app.Stdout.String()).To(MatchRegexp(depRegexp))
+				Expect(app.Stdout.String()).To(MatchRegexp(`Download \[https?://-redacted-:-redacted-.*\]`))
```

Comment:
@jimmykarily  What if we change this to just be 
```
Expect(app.Stdout.String()).To(MatchRegexp(`\[.*-redacted-:-redacted-.*\]`))
```

`libbuildpack` can also `Copy` cached buildpacks instead of `Download`ing them.

Reply:
I missed that case, great!